spectrocloud/cluster_common_tag_map.go

```diff
@@ -0,0 +1,40 @@
+package spectrocloud
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func toTagsMap(d *schema.ResourceData) map[string]string {
+	tags := make(map[string]string)
+	if d.Get("tags_map") != nil {
+		for k, v := range d.Get("tags_map").(map[string]interface{}) {
+			vStr := v.(string)
+			if v != "" {
+				tags[k] = vStr
+			} else {
+				tags[k] = "spectro__tag"
+			}
+		}
+		return tags
+	} else {
+		return nil
+	}
+}
+
+func flattenTagsMap(labels map[string]string) []interface{} {
```

Comment:
@stefanSpectro I can see this function is not utilised in flattening the schema common metadata is still use flattenTags